stdlib/public/SDK/XCTest/XCTestCaseAdditions.mm

```diff
@@ -152,12 +150,11 @@ - (NSString *)className
                    @"reason": exception.reason,
                    };
     }
-
     @catch (...) {
         result = @{
                    @"type": @"unknown",
                    };
     }
 
-    return [result retain];
+    return result;
```

Comment:
I wonder why the original code had `retain` here? I don't know what the requirements on this code are as far as the expected retain convention, but I think this is a behavior change?